### PR TITLE
WMCO 9.0.1 release notes

### PR DESCRIPTION
**Pull request by mburke5678**
Errata: https://errata.devel.redhat.com/docs/show/124870

PEER REVIEW: Only the two final bug texts need review. The first three have been reviewed in multiple other PRs ([10.15.0](https://github.com/openshift/openshift-docs/pull/71436/files), for example). I will update the link to the errata when it is available.

Preview: [Release notes for Red Hat Windows Machine Config Operator 9.0.1](https://72414--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-9-x#wmco-9-0-1)

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-9-x.adoc [hunk 1]
```diff
@@ -26,6 +26,28 @@ You must have this separate subscription to receive support for Windows Containe
 For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[{productwinc}].
 endif::openshift-origin[]
 
+[id="wmco-9-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 9.0.1
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 9.0.1 were released in link:https://access.redhat.com/errata/{replace}[{replace}].
```
Comment (mburke5678) [inline windows_containers/windows-containers-release-notes-9-x.adoc:32]: *@PEER REVIEW* I will update the link when the final URL is available.

**Review by cbippley — COMMENTED** [review]
LGTM, great job!

**Pull request merged**